config/config.h

```diff
@@ -337,8 +337,9 @@
 #define FLATCC_DEFAULT_BIN_SCHEMA_EXT ".bfbs"
 #endif
 
+/* schema file extensions do not carry a dot by convention, do the same here */
```

Comment:
Detail, but other comments start with uppercase and end with '.'

Reply:
I'm fixing it in v5